scalability_and_performance/ztp_far_edge/ztp-advanced-policy-config.adoc

```diff
@@ -40,6 +40,21 @@ include::modules/ztp-configuring-ptp-fast-events.adoc[leveloffset=+1]
 
 * For more information about how to install the AMQ Interconnect Operator, see xref:../../monitoring/using-rfhe.adoc#hw-installing-amq-interconnect-messaging-bus_using-rfhe[Installing the AMQ messaging bus].
 
+include::modules/ztp-add-local-reg-for-sno-duprofile.adoc[leveloffset=+1]
+
+include::modules/ztp-configuring-disk-partitioning.adoc[leveloffset=+2]
+
+include::modules/ztp-configuring-pgt-image-registry.adoc[leveloffset=+2]
+
+[role="_additional-resources"]
+.Additional resources
+
+* For more information about Zero Touch Provisioning (ZTP) setup, see https://docs.openshift.com/container-platform/4.11/scalability_and_performance/ztp-deploying-disconnected.html#about-ztp-and-distributed-units-on-openshift-clusters_ztp-deploying-disconnected[About ZTP and distributed units on OpenShift clusters].
+
+* For more information about the Topology Aware Lifecycle Manager (TALM), see https://docs.openshift.com/container-platform/4.11/scalability_and_performance/cnf-talm-for-cluster-upgrades.html#talo-precache-feature-concept_cnf-topology-aware-lifecycle-manager[Topology Aware Lifecycle Manager for cluster updates].
```

Comment:
Use the following variables for TALM:
{cgu-operator-first} > Topology Aware Lifecycle Manager (TALM)
{cgu-operator-full} > Topology Aware Lifecycle Manager
{cgu-operator} > TALM

Reply:
Done.